src/debug/ee/debugger.cpp

```diff
@@ -15136,7 +15136,8 @@ HRESULT Debugger::InitAppDomainIPC(void)
     } hEnsureCleanup(this);
 
     DWORD dwStrLen = 0;
-    WCHAR szExeName[MAX_LONGPATH];
+    PathString szExeNamePathString;
+    WCHAR * szExeName = szExeNamePathString.OpenUnicodeBuffer(MAX_LONGPATH);
```

Comment:
@jkotas The PathString is InlineSString<512>, and 512 is the size of stack allocated buffer in InlineSString, anything greater than 512 is allocated with new Byte[buffersize] which should be on heap. OpenUnicodeBuffer(bufSize) is the maximum size we expect to write to the buffer, and on CloseBuffer(bufSize), we tell exactly the size to truncate the string. Is my understanding wrong, as far as I saw in code, the OpenUnicodeBuffer(size) does not put the whole buffer on stack.